setup.py

```diff
@@ -1,8 +1,14 @@
+import sys
+
 import setuptools
 
 with open("README.md", "r", encoding="utf8") as fh:
     long_description = fh.read()
 
+onnxruntime_pkg = "onnxruntime"
+if "darwin" == sys.platform and 9 == sys.version_info.minor:
+    onnxruntime_pkg += "<=1.13.1"
```

Comment:
Could you please add annotation why those lines are added?
For the history :slightly_smiling_face:

Reply:
@csh519 
BTW, with #263  it is a good chance to test new release workflow

Reply:
Yup, let's see :)